collections/guides/collect_pii_react/index.md

```diff
@@ -22,7 +22,7 @@ If you'd like to follow along with this guide from scratch, we suggest creating
 
 <span class="base-alert warning">
   <span>
-    To start, you'll need a <code>Elements</code> [Application](https://docs.basistheory.com/api-reference/#applications) with the <code>token:pii:create</code> permission.
+    To start, you'll need a <code>Elements</code> [Application](https://docs.basistheory.com/api-reference/#applications) with the <code>token:pii:create</code> permission. <a href="https://portal.basistheory.com/applications/create?permissions=token%3Apii%3Acreate&type=elements&name=PII+Collector" target="_blank">Click here to create one.</a>
```

Comment:
This deep link is also missing the application type. Do we want to include it so we aren't dependent on the default selection?

Reply:
This one has it. `elements`